dotCMS/src/main/java/com/dotmarketing/util/ImportUtil.java

```diff
@@ -1136,6 +1144,20 @@ else if(UtilMethods.isSet(y)){
 
             for (Contentlet cont : contentlets)
             {
+                int wfActionIdIndex = -1;
+                try {
+                    wfActionIdIndex = Integer.parseInt( results.get( Contentlet.WORKFLOW_ACTION_KEY ).get( 0 ) );
+                } catch ( Exception e ) {
```

Comment:
We could improve the validation in the try in order to avoid exceptions, for example validate if `results.get( Contentlet.WORKFLOW_ACTION_KEY )` is null before to try to get the first element, etc, also we should not have empty catchs, we should have in there at least a log.error